README.md

```diff
@@ -32,7 +32,23 @@ juju deploy sdcore-upf --trust --channel=edge
 
 ### Exposing the UPF externally
 
-If a load balancer such as `metallb` is present, the charm will configure an externally accessible service port with the load balancer upon install of the charm.
+If a load balancer such as `metallb` is present, the charm will configure an externally accessible 
+service port with the load balancer upon install of the charm.
+
+### Running UPF in DPDK mode
+
+By default, UPF runs in `af_packet` mode. To run UPF in `dpdk` mode, `upf-mode` config option 
+should be used, i.e.:
+
+```shell
+juju deploy sdcore-upf --trust --channel=edge --config upf-mode="dpdk" --config enable-hugepages=True --config access-interface-mac-address="00:b0:d0:63:c2:26" --config core-interface-mac-address="00:b0:d0:63:c2:36"
```

Comment:
As we discussed earlier, we need to add a  note that  users need to set following configs according to their environment:
 - core-ip
 - core-gateway-ip
 - access-ip
 - access-gateway-ip

Reply:
I've added the note, but as always, in this kind of cases, I'm not sure about the phrasing :)
@gruyaume would you mind taking a look? Thanks!

Reply:
I think the note is fine, my concern really is more about where this should be documented. I highly prefer placing any requirement in `config.yaml`, closer to the user. See my other comment below.